templates/suspenders_gitignore

```diff
@@ -12,3 +12,4 @@
 /public/assets
 /tags
 /tmp/*
+!.env
```

Comment:
We should remove this line, and the open a PR in our dotfiles **not** to ignore this file.